xml/System/Environment+SpecialFolder.xml

```diff
@@ -647,7 +647,7 @@
       </ReturnValue>
       <MemberValue>45</MemberValue>
       <Docs>
-        <summary>The file system directory that contains the templates that are available to all users. This special folder is valid only for Windows NT systems.  Added in the .NET Framework 4.</summary>
+        <summary>The file system directory that contains the templates that are available to all users. This special folder is valid only for Windows NT systems.</summary>
```

Comment:
[We still support this](https://github.com/dotnet/runtime/search?q=extension%3Acs+CommonTemplates):

```suggestion
        <summary>The file system directory that contains the templates that are available to all users.</summary>
```